src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -79,7 +81,7 @@ public static void WarmUpCacheInCurrentProcess(Project project)
             else
             {
                 result = await GetUnimportedExtensionMethodsInCurrentProcessAsync(
-                    document, position, receiverTypeSymbol, namespaceInScope, targetTypesSymbols, forceCacheCreation, hideAdvancedMembers, remoteAssetSyncTime: null, cancellationToken)
+                    document, syntaxContext.SemanticModel, position, receiverTypeSymbol, namespaceInScope, targetTypesSymbols, forceCacheCreation, hideAdvancedMembers, remoteAssetSyncTime: null, cancellationToken)
```

Comment:
could we just pass syntaxContext into this method?  given that we're passing doc/semanticmodel/pos?

Reply:
❓ Any objection to keeping the current form in light of https://github.com/dotnet/roslyn/pull/71872#discussion_r1475251037? It seemed like forcing the calculation of a complete `SemanticContext` might be overkill.